### PR TITLE
[zh] resync page in scheduling-eviction

### PR DESCRIPTION
**Pull request by Zhuzhenghao**
SInce the PR changed a lot of files, I seperate them into two PR, the other one is #39677.

This PR resync these pages:

- `content/zh-cn/docs/concepts/scheduling-eviction/kube-scheduler.md`
- `content/zh-cn/docs/concepts/scheduling-eviction/pod-priority-preemption.md`
- `content/zh-cn/docs/concepts/scheduling-eviction/taint-and-toleration.md`

### CODE REVIEW

**Review by Sea-n — COMMENTED** [review]
/lgtm

File: content/zh-cn/docs/concepts/scheduling-eviction/pod-priority-preemption.md [hunk 1]
```diff
@@ -484,20 +485,20 @@ enough demand and if we find an algorithm with reasonable performance.
 只有当另一个节点上的 Pod 被抢占时，P 才可能在 N 上变得可行。
 下面是一个例子：
 
-*   正在考虑将 Pod P 调度到节点 N 上。
-*   Pod Q 正在与节点 N 位于同一区域的另一个节点上运行。
-*   Pod P 与 Pod Q 具有 Zone 维度的反亲和（`topologyKey:topology.kubernetes.io/zone`）。
-*   Pod P 与 Zone 中的其他 Pod 之间没有其他反亲和性设置。
-*   为了在节点 N 上调度 Pod P，可以抢占 Pod Q，但调度器不会进行跨节点抢占。
-    因此，Pod P 将被视为在节点 N 上不可调度。
+* 正在考虑将 Pod P 调度到节点 N 上。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/pod-priority-preemption.md:488]: 無主句？要不要加上主語“調度器”？

File: content/zh-cn/docs/concepts/scheduling-eviction/pod-priority-preemption.md [hunk 2]
```diff
@@ -484,20 +485,20 @@ enough demand and if we find an algorithm with reasonable performance.
...
+* Pod Q 正在与节点 N 位于同一区域的另一个节点上运行。
+* Pod P 与 Pod Q 具有 Zone 维度的反亲和（`topologyKey:topology.kubernetes.io/zone`）。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/pod-priority-preemption.md:490]:
```suggestion
* Pod P 与 Pod Q 具有 Zone 维度的反亲和（`topologyKey:topology.kubernetes.io/zone`）设置。
```

File: content/zh-cn/docs/concepts/scheduling-eviction/pod-priority-preemption.md (hunk 1 above)
Comment (Zhuzhenghao) [reply content/zh-cn/docs/concepts/scheduling-eviction/pod-priority-preemption.md:488]: 赞成，我加上了主语

**Comment by tengqm** [comment]
Please rebase this onto 'main' and recommit.

**Comment by tengqm** [comment]
@Zhuzhenghao This one needs a rebase.

**Comment by Zhuzhenghao** [comment]
Thanks for reminding, Rebased.

File: content/zh-cn/docs/concepts/scheduling-eviction/pod-priority-preemption.md [hunk 3]
```diff
@@ -486,20 +486,21 @@ enough demand and if we find an algorithm with reasonable performance.
...
-*   Pod P 与 Zone 中的其他 Pod 之间没有其他反亲和性设置。
-*   为了在节点 N 上调度 Pod P，可以抢占 Pod Q，但调度器不会进行跨节点抢占。
-    因此，Pod P 将被视为在节点 N 上不可调度。
+- 调度器正在考虑将 Pod P 调度到节点 N 上。
+- Pod Q 正在与节点 N 位于同一区域的另一个节点上运行。
+
+* Pod P 与 Pod Q 具有 Zone 维度的反亲和（`topologyKey:topology.kubernetes.io/zone`）设置。
+* Pod P 与 Zone 中的其他 Pod 之间没有其他反亲和性设置。
+* 为了在节点 N 上调度 Pod P，可以抢占 Pod Q，但调度器不会进行跨节点抢占。
+  因此，Pod P 将被视为在节点 N 上不可调度。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/scheduling-eviction/pod-priority-preemption.md:495]:
```suggestion
* 调度器正在考虑将 Pod P 调度到节点 N 上。
* Pod Q 正在与节点 N 位于同一区域的另一个节点上运行。
* Pod P 与 Pod Q 具有 Zone 维度的反亲和（`topologyKey:topology.kubernetes.io/zone`）设置。
* Pod P 与 Zone 中的其他 Pod 之间没有其他反亲和性设置。
* 为了在节点 N 上调度 Pod P，可以抢占 Pod Q，但调度器不会进行跨节点抢占。
  因此，Pod P 将被视为在节点 N 上不可调度。
```

**Review by windsonsea — COMMENTED** [review]
Pick for consistency nearby.

File: content/zh-cn/docs/concepts/scheduling-eviction/kube-scheduler.md [hunk 4]
```diff
@@ -104,15 +104,15 @@ locality, inter-workload interference, and so on.
 亲和以及反亲和要求、数据局部性、负载间的干扰等等。
 
 <!--
-## Scheduling with kube-scheduler {#kube-scheduler-implementation}
+### Node selection in kube-scheduler {#kube-scheduler-implementation}
 -->
-## kube-scheduler 调度流程 {#kube-scheduler-implementation}
+## kube-scheduler 中的节点选择 {#kube-scheduler-implementation}
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/scheduling-eviction/kube-scheduler.md:109]:
```suggestion
## kube-scheduler 中的节点选择 {#kube-scheduler-implementation}
```
这个 heading 的层级跟 en 不一致

**Pull request merged**